src/riscv-hybrid-integration.adoc

```diff
@@ -452,9 +437,9 @@ xref:menvcfgmodereg[xrefstyle=short].
 include::img/menvcfgmodereg.edn[]
 
 The CHERI Register Enable (CRE) bit controls whether less privileged levels can
-perform explicit accesses to CHERI registers.  When CRE=1, CHERI registers can
-be read and written by less privileged levels.  When CRE=0, CHERI registers are
-disabled in less privileged levels as described in
+perform explicit accesses to CHERI registers. When CRE=1 and <<mseccfg>>.CRE=1,
+CHERI registers can be read and written by less privileged levels. When CRE=0,
+CHERI registers are disabled in less privileged levels as described in
```

Comment:
```suggestion
perform explicit accesses to CHERI registers. perform explicit accesses to CHERI registers. When the CRE bit is set to 1 and <<mseccfg>>.CRE=1,
CHERI registers can be read and written by less privileged levels. When set to 0,
CHERI registers are disabled in less privileged levels as described in
```

I briefly got confused here whether we are talking about the effective CRE or the CRE bit here, maybe worth being explicit?